README.md

```diff
@@ -79,309 +24,324 @@ The better results with auto-labeling were achieved by fine-tuning a ViT-H+ on t
+| LightlyTrain | dinov3/vits16-eomt-ade20k | ✅ | **0.533** | 21.6 | 518×518 |
+| LightlyTrain | dinov3/vitb16-eomt | ❌ | 0.544 | 85.7 | 518×518 |
+| LightlyTrain | dinov3/vitb16-eomt-ade20k | ✅ | **0.573** | 85.7 | 518×518 |
 
-<details>
-<summary><strong>Why should I use LightlyTrain instead of other already pretrained models?</strong></summary>
+The better results with auto-labeling were achieved by fine-tuning a ViT-H+ on the
+ADE20K dataset, which reaches 0.595 validation mIoU. This model was then used to autolabel
+100k images from the SUN397 dataset. Using these labels, we subsequently fine-tuned the
+smaller models, and then used the ADE20k dataset for validation.
+
+#### Usage
 
-LightlyTrain is most beneficial when:
+[![Documentation](https://img.shields.io/badge/Documentation-blue)](https://docs.lightly.ai/train/stable/predict_autolabel.html)
 
-- **Working with domain-specific data**: When your data has a very different distribution from standard datasets (medical images, industrial data, etc.)
-- **Facing policy or license restrictions**: When you can't use models pretrained on datasets with unclear licensing
-- **Having limited labeled data**: When you have access to a lot of unlabeled data but few labeled examples
-- **Using custom architectures**: When no pretrained checkpoints are available for your model
+```python
+import lightly_train
 
-LightlyTrain is complementary to existing pretrained models and can start from either random weights or existing pretrained weights.
+if __name__ == "__main__":
+  lightly_train.predict_semantic_segmentation(
+    out="out/my_autolabeled_data",
+    data="my_data_dir",
+    model="dinov3/vitb16-eomt-ade20k",
+  )
+```
 
 </details>
 
-Check our [complete FAQ](https://docs.lightly.ai/train/stable/faq.html) for more information.
+## Features
+
+- Python, Command Line, and [Docker](https://docs.lightly.ai/train/stable/docker.html) support
+- Built for [high performance](https://docs.lightly.ai/train/stable/performance/index.html) including [multi-GPU](https://docs.lightly.ai/train/stable/performance/multi_gpu.html) and [multi-node](https://docs.lightly.ai/train/stable/performance/multi_node.html) support
+- [Monitor training progress](https://docs.lightly.ai/train/stable/train.html#loggers) with MLflow, TensorBoard, Weights & Biases, and more
+- Runs fully on-premises with no API authentication
+- Export models in their native format for fine-tuning or inference
+- Export models in ONNX or TensorRT format for edge deployment
+
+## Models
+
+LightlyTrain supports the following model and workflow combinations.
+
+Scroll right to see all columns!
+
+| Framework | Model | Object Detection<br><sub>(*Labeled Images*)</sub> | Instance Segmentation<br><sub>(*Labeled Images*)</sub> | Semantic Segmentation<br><sub>(*Labeled Images*)</sub> | Distill From<br> DINOv2/v3<br><sub>(*Unlabeled Images*)</sub> | Pretrain<br><sub>(*Unlabeled Images*)</sub> | Autolabel |
```

Comment:
It's a cool idea with the table but this is too verbose and uses too much space for little information. I'd leave it away